src/client/components/Appheader/AppHeader.component.js

```diff
@@ -1,15 +1,17 @@
 import './AppHeader.style.css';
 
 import React from 'react';
+import { useUser } from '../../context/userContext';
+import { Button } from '../Button/Button';
 
 import Vector from './Vector.svg';
 
 export const AppHeader = () => {
+  const { isAuthenticated } = useUser();
```

Comment:
Are you sure storybook works after this addition?

Reply:
I did check, storybook is running well